app/components/markdown/markdown_table/markdown_table.js

```diff
@@ -54,7 +54,8 @@ export default class MarkdownTable extends React.PureComponent {
         });
         const passProps = {
             renderRows: this.renderRows,
-            tableWidth: this.getTableWidth(),
+            tableWidth: this.getTableWidth(true),
+            renderAsFlex: this.shouldRenderAsFlex(true),
```

Comment:
When rotating the device this prop won't get updated.

Reply:
I've been taking a closer look at this, and while it's true that this prop won't get updated on device rotation, the behavior of tables (whether to render as flexbox, or calculate a fixed width) isn't affected by whether the user is in landscape mode or not

The landscape bit is (currently) only being used on 4-column tables in the channel view, as they should render as a flexbox in landscape mode but otherwise should have a fixed width when in portrait mode. 

So I'm not so sure if using `onLayout` to run another check on whether or not to render as flex would change much here?

Reply:
I think you are right, lets keep it as is and in any case we could look for an alternative later on